projects/1.19/assets/quark/quark/lang/zh_cn.json

```
@@ -1,1296 +1,3379 @@
+	"advancements.quark.tame_foxhound.title": "玩火",
+	"advancements.quark.tame_foxhound.description": "在防火效果下喂食煤炭来驯服一只猎狐犬",
+	"advancements.quark.foxhound_speed_up_furnace.title": "让猎狐犬躺下",
+	"advancements.quark.foxhound_speed_up_furnace.description": "让猎狐犬在熔炉上睡觉来加快熔炉的速度",
+	"advancements.quark.shiba_help.title": "给小镇上色",
+	"advancements.quark.shiba_help.description": "手持火把，在柴犬的帮助下清理一个暗处",
+	"advancements.quark.poison_baby.title": "青春永驻",
+	"advancements.quark.poison_baby.description": "用毒马铃薯喂食一只幼年动物，使其无法成长",
+	"advancements.quark.crab_in_a_bucket.title": "螃蟹捕获",
+	"advancements.quark.crab_in_a_bucket.description": "用桶捕获一只螃蟹",
+
+
+
+	"_comment": "配置翻译，置于文件末尾，便于未来比对更新非配置部分。",
+	"quark.config.general.q_button_on_the_right.name": "「q」按钮位于右侧",
+
+	"quark.config.general.advancement_visibility_depth.name": "进度可见深度",
+	"quark.config.general.advancement_visibility_depth.desc": "你能在进度界面中看到多深的进度。原版为2。\n允许的值：(0,)",
+
+	"quark.config.general.allowed_screens.name": "可显示按钮的界面",
+	"quark.config.general.allowed_screens.desc": "能接受夸克按钮的界面列表。\n使用「输出界面类名」找到你想添加的其他界面的名字。",
+
+	"quark.config.general.disable_q_menu_effects.name": "禁用「q」菜单特效",
+	"quark.config.general.disable_q_menu_effects.desc": "进入夸克配置主界面时不再快速旋转与模糊。",
+
+	"quark.config.general.enable_q_button.name": "启用「q」按钮",
+	"quark.config.general.enable_q_button.desc": "方便你不经过模组列表快速配置夸克。",
+
+	"quark.config.general.enable_advancement_modification.name": "启用进度修改",
+	"quark.config.general.enable_advancement_modification.desc": "夸克会向原版的多要求进度中新增要求，设为false禁用。",
+
+	"quark.config.general.enable_jei_item_info.name": "启用JEI物品信息",
+	"quark.config.general.enable_jei_item_info.desc": "使用JEI查看物品的配方/用途时显示夸克的物品信息，设为false禁用。",
+
+	"quark.config.general.enable_network_profiling.desc": "启用网络分析\n\n启用夸克网络分析功能。没被要求就不要启用它。",
+
+	"quark.config.general.enable_onboarding.name": "启用入门引导",
+	"quark.config.general.enable_onboarding.desc": "弹出气泡信息告诉你可以在「q」菜单中编辑夸克配置，设为false禁用。",
+
+	"quark.config.general.enable_quark_advancements.name": "启用夸克进度",
+	"quark.config.general.enable_quark_advancements.desc": "夸克会添加自己的进度，设为false禁用。",
+
+	"quark.config.general.enable_worldgen_watchdog.desc": "启用世界生成看门狗\n\n设为true后，会启用调试系统，用于调试夸克的世界生成功能。\n此功能§c只§r应在开发人员要求时使用。",
+
+	"quark.config.general.hide_disabled_content.name": "隐藏被禁用的内容",
+	"quark.config.general.hide_disabled_content.desc": "夸克会自动隐藏任何被禁用的物品，设为false禁用此行为。",
+
+	"quark.config.general.interpret_shulker_box_like_blocks.name": "显示类潜影盒方块的内容物",
+	"quark.config.general.interpret_shulker_box_like_blocks.desc": "夸克是否应该将任何ID中带有「shulker_box」的物品都视为潜影盒？",
+
+	"quark.config.general.piston_push_limit.name": "活塞推动上限",
+	"quark.config.general.piston_push_limit.desc": "活塞能一次推动的最大方块数量。\n允许的值：(0,)",
+
+	"quark.config.general.print_screen_classnames.name": "输出界面类名",
+	"quark.config.general.print_screen_classnames.desc": "需要找到有问题的界面的类名时，将它设为true。",
+
+	"quark.config.general.shulker_boxes.name": "潜影盒",
+	"quark.config.general.shulker_boxes.desc": "应被夸克视为潜影盒的方块。",
+
+	"quark.config.general.suppressed_info.name": "被压制的信息",
```

```suggestion
	"quark.config.general.suppressed_info.name": "需要抑制的信息",
```
理论上再加一个JEI在“信息”前会更完整